contrib/sarplus/python/pysarplus/SARPlus.py

```diff
@@ -381,9 +342,7 @@ def recommend_k_items_slow(self, test, top_k=10, remove_seen=True):
         if remove_seen:
             raise ValueError("Not implemented")
 
-        self.get_user_affinity(test).write.mode("overwrite").saveAsTable(
-            self.f("{prefix}user_affinity")
-        )
+        self.get_user_affinity(test).write.mode("overwrite").saveAsTable(self.f("{prefix}user_affinity"))
```

Comment:
It may be nice to apply black formatting to these files.

Reply:
Done

Reply:
https://github.com/microsoft/recommenders/pull/1566/commits/466ebc83db626110c533cf3e54b9cec319e8e508